gnomock.go

```diff
@@ -217,14 +217,37 @@ func Stop(cs ...*Container) error {
 		container := c
 
 		eg.Go(func() error {
-			return g.stop(container)
+			return g.stop(context.Background(), container)
 		})
 	}
 
 	return eg.Wait()
 }
 
-func (g *g) stop(c *Container) error {
+// StopCtx is the same as Stop, but it allows to provide a context to stop
+// containers.
+func StopCtx(ctx context.Context, cs ...*Container) error {
+	g, err := newG(isInDocker())
+	if err != nil {
+		return err
+	}
+
+	defer func() { _ = g.log.Sync() }()
+
+	eg, eCtx := errgroup.WithContext(ctx)
```

Comment:
I think that we don't need two copies of the same function. Instead, we need the `Stop` function to call `StopCtx` with the background context, and `StopCtx` should be the only function that has all the errgroup code.

Reply:
Yes, I've checked that `github.com/docker/docker/client.ContainerStop` and `github.com/docker/docker/client.ContainerRemove` both take the same context. Though, I didn't this behavior specifically.